components/Common/AnimatedPlaceholder/index.module.scss

```diff
@@ -0,0 +1,47 @@
+.animatedBackground,
+%animated-background {
+  animation-duration: 1.25s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: infinite;
+  animation-name: placeHolderShimmer;
+  animation-timing-function: linear;
+  background: #f6f6f6;
+  background: linear-gradient(to right, #f6f6f6 8%, #f0f0f0 18%, #f6f6f6 33%);
+  background-size: 800px 104px;
+  height: 96px;
+  position: relative;
+}
+
+.placeholder {
+  display: flex;
+  width: 100%;
+
+  @keyframes placeHolderShimmer {
```

Comment:
Can we remove the keyframes from this level? As keyframes in theory should not be contained within a class?